build/ci/evergreen.yml

```diff
@@ -794,27 +794,6 @@ tasks:
           MCLI_OPS_MANAGER_URL: ${mcli_ops_manager_url}
           MCLI_SERVICE: cloud
           E2E_TAGS: atlas,networking
-  # datalake requires cloud provider role authentication and an s3 bucket created
```

Comment:
Do we want to remove these tests even if the `mongocli atlas datalake` won't be deprecated?

Reply:
mongocli atlas as a whole is deprecated, it should have been cleaned up by end of April, we just need to schedule that work.

Keeping these tests would require some changes on our E2E to ignore deprecation messages, so that would be wasted effort IMO.